packages/contracts-bedrock/contracts/deployment/BaseSystemDictator.sol

```diff
@@ -1,14 +1,76 @@
 // SPDX-License-Identifier: MIT
 pragma solidity 0.8.15;
 
+import { ProxyAdmin } from "../universal/ProxyAdmin.sol";
+import { L2OutputOracle } from "../L1/L2OutputOracle.sol";
+import { OptimismPortal } from "../L1/OptimismPortal.sol";
+import { L1CrossDomainMessenger } from "../L1/L1CrossDomainMessenger.sol";
+import { L1StandardBridge } from "../L1/L1StandardBridge.sol";
+import { L1ERC721Bridge } from "../L1/L1ERC721Bridge.sol";
+import { OptimismMintableERC20Factory } from "../universal/OptimismMintableERC20Factory.sol";
+import { AddressManager } from "../legacy/AddressManager.sol";
+import { PortalSender } from "./PortalSender.sol";
 import { Ownable } from "@openzeppelin/contracts/access/Ownable.sol";
-import { SystemConfig } from "./SystemConfig.sol";
 
 /**
  * @title BaseSystemDictator
  * @notice The BaseSystemDictator is a base contract for SystemDictator contracts.
  */
 contract BaseSystemDictator is Ownable {
+    /**
```

Comment:
Moved this into here because otherwise I would get an error because `SystemConfig.sol` doesn't have a buildInfo file.

Reply:
Ok i'm into the idea of moving them into this file